app/graphql/mutations/create_budget.rb

```diff
@@ -5,16 +5,21 @@ class Mutations::CreateBudget < Mutations::BaseMutation
   argument :amount, Float, required: true
 
   field :user_id, ID, null: false
-  field :month, String, null: true
-  field :category, String, null: true
-  field :amount, Float, null: true
+  field :budget, Types::BudgetType, null: false
```

Comment:
We discussed this on call yesterday, but yes; I think this totally makes sense to have created objects' fields nested under their respective objects.